install.sh

```diff
@@ -17,14 +17,29 @@ wget -q https://git.io/voEUQ -O /tmp/raspap && bash /tmp/raspap -y -o 0
 
 echo 'Configuring network interfaces to static'
 # Set static IP for eth0 and wlan0
-sudo cp ./templates/interfaces.template /etc/network/interfaces
 
-# Let raspAP know about eth0 config
-sudo cp ./templates/eth0.ini.template /etc/raspap/networking/eth0.ini 
+#Get the dhcp info and set it as static
+staticip=`ip addr show eth0 | grep -Po "inet \d+\.\d+\.\d+\.\d+" | sed "s/inet //"`
+gateway=`ip route | grep default | grep -Po -m 1 "\d+\.\d+\.\d+\.\d+" | head -1`
+network=`echo $staticip | sed "s/\([0-9]\+\.[0-9]\+\.[0-9]\+\.\)[0-9]\+/\10/"`
 
-# Let raspAP know about wlan0 config
-sudo cp ./templates/wlan0.ini.template /etc/raspap/networking/wlan0.ini
+# Set static IP for eth0 and wlan0
+while read line
+do
+        echo $line | sed "s/192.168.1.1/$gateway/g" | sed "s/192.168.1.0/$network/g" | sed "s/192.168.1.99/$staticip/g"
+done < ./templates/interfaces.template > /etc/network/interfaces
 
+# Let rapAP know about eth0
+while read line
+do
+        echo $line | sed "s/192.168.1.1/$gateway/g" | sed "s/192.168.1.0/$network/g" | sed "s/192.168.1.99/$staticip/g"
+done < ./templates/eth0.ini.template > /etc/raspap/networking/eth0.ini
+
+# Let rapAP know about eth0
+while read line
```

Comment:
I'm only changing the router gateway and dns server. Technically I can take out the static IP sed, but since it's already different than before, no need.

Reply:
Are you not assigning the exact same values to wlan0 that you had for eth0?

Also, that sed wont work because those patterns don't exist in wlan0.ini.template

Reply:
I can edit out the sed that does network and static ip. It only matches the gateway, which is still the same since the wlan0 gateway is the same as the eth0 gateway

Reply:
Correcting myself: the first sed will work to replace the gateway and dns, but the other two shouldn't be there if we know they wont match anything